my-app/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./reset.css";
+import "./App.css";
+import Nav from "./components/Nav.js";
+import Post from "./components/Post.js";
+import Posts from "./components/Posts.js";
+import More from "./components/More.js";
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = { posts: [], currentPage: 0 };
+  }
+
+  moreBtnClickHandler = e => {
+    this.setState(currentState => {
+      return {
+        posts: currentState.posts,
+        currentPage: ++currentState.currentPage
+      };
+    });
+  };
+
+  componentDidMount() {
+    this.fetchTopStories();
+  }
+
+  fetchTopStories() {
+    fetch("https://hacker-news.firebaseio.com/v0/topstories.json")
```

Comment:
Would be great to see some error handling in case the network request fails

Reply:
Also, just curious: you seem to import some firebase stuff both in your package.json as well as the index.html - any reason why this is not used here?